avro2tf/src/test/resources/data/share_feat_list/feat_lists_case_2/fruits_day2

```diff
@@ -2,5 +2,6 @@ tue_fruits,pear
 tue_fruits,apple
 tue_fruits,banana
 tue_fruits,kiwi
+tue_fruits,apple,123
```

Comment:
Please also add some special characters like @ $ . % ... to terms and make sure it works as expected

Reply:
done. Thanks